src/modules/getScores.js

```diff
@@ -0,0 +1,20 @@
+const scoreUl = document.querySelector('.scores');
+
+async function getScores() {
+  const res = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/IqNrWE04pgaXdxjtzDh9/scores/');
+  const resj = await res.text();
+  const scores = JSON.parse(resj).result;
```

Comment:
[OPTIONAL] As we are still trying to get familiar with the es6 syntax, this would be been a nice place to showcase your understanding of the `destructuring` assignment. I suggest that you destructure the properties from the response instead of using the dot operator. 

like this 

```
 const { result } = await res.text();
 const scores = JSON.parse(result);
```

You notice the logic looks cleaner right 😄? 

Consider this approach to make your code more readable.